core/src/main/java/org/infinispan/commands/read/KeySetCommand.java

```diff
@@ -48,8 +48,8 @@ public Object acceptVisitor(InvocationContext ctx, Visitor visitor) throws Throw
    }
 
    @Override
-   public boolean readsExistingValues() {
-      return false;
+   public LoadType loadType() {
+      return null;
```

Comment:
I would say `UNDEF` - yep, I could add that. Since you didn't like the `OTHER` option in the past, I've rather used null :)